src/agent/debuggable-module/src/windows.rs

```diff
@@ -141,6 +141,7 @@ impl<'data> WindowsModule<'data> {
                 let mut symbols = mi.symbols()?;
 
                 while let Some(symbol) = symbols.next()? {
+                    #[allow(clippy::single_match)]
```

Comment:
We will hopefully reintroduce label scrutiny later. Suppressing this to minimize churn, and frankly, I don't think and `if let` would be more readable anyway.